index.html

```diff
@@ -133,10 +141,22 @@ <h4 class="list-title">Skills</h4>
     <h3 class="contact-header">Get started</h3>
     <p class="contact-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
     <form class="form" action="https://formspree.io/f/mjvzkaal" method="post">
-      <input type="text" maxlength="30" placeholder="Your name" class="form-input" name="name" required />
-      <input type="email" placeholder="Your email" name="email" class="form-input" required />
-      <textarea maxlength="500" placeholder="Enter your message here..." name="message" class="form-text" required></textarea>
-      <button type="submit" class="form-btn">Start collaboration</button>
+      <div class="det-cont">
+        <div class="name-cont">
+          <label>Your name</label>
+          <input type="text" maxlength="30" placeholder="Your name" class="form-input" name="name" required />
+        </div>
+        <div class="name-cont">
+          <label>Your name</label>
+          <input type="email" placeholder="Your email" name="email" class="form-input" required />
+        </div>
+      </div>
+      <div class="text-cont-3">
+        <textarea maxlength="500" placeholder="Enter your message here..." name="message" class="form-text" required></textarea>
+      </div>
+      <div class="btn-cont">
+        <button type="submit" class="form-btn">Start collaboration</button>
+      </div>
```

Comment:
**kindly fix those issues to match the Figma design :**

- [ ] you have a missing background image in the bottom right corner, kindly add it like shown in Figma
- [ ] you have a missing message label before the `textarea `, kindly add it to guide the users when they want to contact you
- [ ] please add the background color `#F9F9FA` for this section to match figma design
- [ ] try to update your input styles by following the padding and margins and font size so it matches figma


******************


- **_your design_**

![image](https://user-images.githubusercontent.com/89282221/187047765-4471b326-c6ab-4d12-9640-f80e159f32c4.png)

- **_figma design_**


![image](https://user-images.githubusercontent.com/89282221/187047802-ff0f1fce-b184-4b2a-9984-786c1bdc1b2f.png)

Reply:
Kindly, I recommend you to follow the suggestions of the previous reviewer, this will help your project to look more PRO 😉